modules/nextflow/src/main/groovy/nextflow/trace/WorkflowStats.groovy

```diff
@@ -114,22 +114,17 @@ class WorkflowStats implements Cloneable {
     }
 
     float getSucceedPct() {
-        int tot = gtz(succeededCount + cachedCount + ignoredCount + failedCount)
+        int tot = gtz(succeededCount + cachedCount + failedCount)
```

Comment:
This total should include the aborted count too right? Since this total is used four times I would just factor it into a separate `getTotalCount()` function